python/ray/experimental/workflow/serialization.py

```diff
@@ -0,0 +1,94 @@
+import asyncio
+from dataclasses import dataclass
+from enum import Enum
+import ray
+from ray.types import ObjectRef
+from ray.experimental.workflow.common import calculate_identifier
+from ray.experimental.workflow.storage import Storage
+from ray.experimental.workflow import workflow_storage
+from typing import Any, Dict, List, Optional, Tuple, TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from ray.actor import ActorHandle
+
+MANAGEMENT_ACTOR_NAME = "StorageManagementActor"
+MANAGEMENT_ACTOR_NAMESPACE = "workflow"
```

Comment:
To manage them together with workflow_access.py? I mean we have two actors now, and probably it's better to have a centralized place for all the namings.